semver/mod.ts

```diff
@@ -274,6 +274,14 @@ export interface Options {
   includePrerelease?: boolean;
 }
 
+export interface SemVer {
+  major: number;
+  minor: number;
+  patch: number;
+  prerelease: ReadonlyArray<string | number>;
+  build: ReadonlyArray<string>;
+}
+
 // Note: this is the semver.org version of the spec that it implements
 // Not necessarily the package version of this code.
 export const SEMVER_SPEC_VERSION = "2.0.0";
```

Comment:
Why do we export the spec version? Shouldn‘t a comment be enough?

Reply:
I have no idea, I'll look to see if there are references and then clean this up. I'm going to have a little room to take this back off the shelf today.